hivemind/p2p/servicer.py

```diff
@@ -24,9 +24,10 @@ class StubBase:
     adding the necessary rpc_* methods. Calls to these methods are translated to calls to the remote peer.
     """
 
-    def __init__(self, p2p: P2P, peer: PeerID):
+    def __init__(self, p2p: P2P, peer: PeerID, namespace: Optional[str]):
```

Comment:
```suggestion
    def __init__(self, p2p: P2P, peer: PeerID, namespace: Optional[str]=None):
```

Reply:
I'd suggest to avoid setting default parameter values in functions that should not be directly called by user (this constructor is one of them because the stub should be instantiated via `Servicer.get_stub()`).

This is to (1) avoid forgetting to pass a non-default value in the library code and (2) avoid duplicating defaults.